sycl/include/sycl/ext/intel/experimental/esimd/esimd.hpp

```diff
@@ -352,31 +362,41 @@ template <typename Ty, int N> class simd {
   /// \param Offset is offset in number of elements in src region.
   /// \return replicated simd instance.
   template <int Rep, int VS, int W>
+  __SYCL_DEPRECATED("use simd::replicate_vs_w")
   simd<Ty, Rep * W> replicate(uint16_t Offset) {
-    return replicate<Rep, VS, W, 1>(Offset);
+    return replicate_vs_w<Rep, VS, W>(Offset);
+  }
+
+  /// \tparam Rep is number of times region has to be replicated.
+  /// \tparam VS vertical stride of src region to replicate.
+  /// \tparam W is width of src region to replicate.
```

Comment:
Nit: here and in other places. 'is' would seem redundant in generated docs.
```suggestion
  /// \tparam W width of src region to replicate.
```